File: christmas/december task 2/Square.java
import java.util.Scanner;
    public class Square { 
        public static void main(String...Odili) {
            Scanner input = new Scanner(System.in);
            int number = 0;
            System.out.println(isSquare(number));
        }

        public static boolean isSquare(int number){
            Scanner input = new Scanner(System.in);

        System.out.println("Enter any integer ");
            number = input.nextInt();
                int result = 0;
    
            for(int count = 1; count <= number; count ++){     
                if(count * count  == number){
                    return true;
                }     
          
             }

            return false;
        }
    }

File: christmas/december task 2/square.py
def isSquare(number):
    number = int(input("Enter any integer "))
           
    result = 0
    
    for count in range (1, number + 1):     
        if(count * count  == number):
            return True
        

        return False
      
    
number = 0
print(isSquare(number))
 
